projects/ccf-rui/src/themes/_light.scss

```diff
@@ -35,6 +35,8 @@ $foreground: map-merge($mat-light-theme-foreground, (
   slider-green: map-get($mat-green, A700), // #00C853
   slider-red: map-get($mat-red, A700), // #D50000
   text: map-get($primary-colors, 900), // #212121
+  toggle-bar: rgba(25, 118, 210, 0.54),
```

Comment:
What are these colors? Don't just move them! Instead use the correct `map-get(...)`. As for the opacity channel it should be applied in the component's theme file using the second for of `mat-color(from, variable, opacity)`